src/libMessage/Messenger.cpp

```diff
@@ -5616,19 +5636,34 @@ bool Messenger::SetLookupGetStartPoWFromSeed(bytes& dst,
 
 bool Messenger::GetLookupGetStartPoWFromSeed(const bytes& src,
                                              const unsigned int offset,
-                                             uint32_t& listenPort) {
+                                             uint32_t& listenPort,
```

Comment:
need to pass pubkey to the caller function. The caller function need to ensure the pubkey is known.